src/components/ApiAction/SaveQueryModal.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <b-modal
+    id="modal-save-query"
+    title="Choose a name for that query"
```

Comment:
```suggestion
    title="Choose a name for this query"
```

Reply:
Forgot to update `query` to `API Action`